controllers/tagController.js

```diff
@@ -0,0 +1,38 @@
+const tags = require('../models/tags');
+
+// an onclick function will place the entry_id and the specific tag that was clicked on in the req body
+const add = (req,res) =>{
+    const { tag,entry_id } = req.body
```

Comment:
These look great! Be sure to write what the routes that will use these controllers should look like!